internal/distro/rhel9/imagetype.go

```diff
@@ -257,6 +257,8 @@ func (t *imageType) Manifest(bp *blueprint.Blueprint,
 	}
 
 	manifest.Content.PackageSets = manifest.GetPackageSetChains()
+	manifest.Content.Containers = manifest.GetContainerSourceSpecs()
+	manifest.Content.OSTreeCommits = manifest.GetOSTreeSourceSpecs()
```

Comment:
I'm wondering what's the advantage of having both, the `GetXXXSourceSpecs()` method and exported `manifest.Content.XXX` provided and containing the same value at the same time? 🤔  Shouldn't this be abstracted from the user of `manifest.Manifest` in its implementation?